.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Subject of the issue
```

Comment:
Can we only issue a bugs or are we able to issue also patches and features in here?

Reply:
I went with the GitHub definition of an issue. Which means an issue can be a bug or feature ;)